drupal/patches/d10_version_support.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/translatable_menu_link_uri.info.yml b/translatable_menu_link_uri.info.yml
```

Comment:
This module has D10 support now in 2.1.0 version https://www.drupal.org/project/translatable_menu_link_uri

Reply:
I guess the this file can be removed?